packages/fixtures/src/working-groups.ts

```diff
@@ -58,10 +60,13 @@ export const createWorkingGroupResponse = (
   shortText: `Working Group ${itemIndex} Short Text`,
   lastModifiedDate: '2020-11-09T20:36:54Z',
   externalLink: `https://www.example.com/working-group-${itemIndex}`,
-  deliverables: createDeliverables(options?.deliverables ?? 1),
+  deliverables: createDeliverables(deliverables ?? 1),
   complete: false,
-  members: createWorkingGroupMembers(options?.members ?? 1),
-  leaders: createWorkingGroupLeaders(options?.members ?? 1),
+  members: createWorkingGroupMembers(members ?? 1),
+  leaders: createWorkingGroupLeaders(members ?? 1),
+  calendars: Array.from({ length: calendarsCount }, (_, index) =>
```

Comment:
I don't think you will get your default if you set deliverables to 1 because calendars count will be undefined. If thats not the case can you change teh others to be consistent with this approach